packages/clay-navigation-bar/src/Hooks.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This file should be lowercase since it isnt exporting a component by default. `hooks.ts`